lib/msf/core/modules/loader/directory.rb

```diff
@@ -42,7 +42,7 @@ def each_module_reference_name(path, opts={})
           entry_descendant_pathname = Pathname.new(entry_descendant_path)
           relative_entry_descendant_pathname = entry_descendant_pathname.relative_path_from(full_entry_pathname)
           relative_entry_descendant_path = relative_entry_descendant_pathname.to_s
-          next if File::basename(relative_entry_descendant_path) == "example.rb"
+          next if ['example.rb', 'example_linux_priv_esc.rb', 'example_webapp.rb'].include? File::basename(relative_entry_descendant_path)
```

Comment:
Instead of adding more cases, we should make the check slightly more generic.

```suggestion
          next if File::basename(relative_entry_descendant_path).start_with?('example')
```